apps/tlon-mobile/src/components/AuthenticatedApp.tsx

```diff
@@ -0,0 +1,36 @@
+import { ZStack } from '@tloncorp/ui';
+
+import { WebviewPositionProvider } from '../contexts/webview/position';
+import { WebviewProvider } from '../contexts/webview/webview';
+import { useDeepLinkListener } from '../hooks/useDeepLinkListener';
+import useNotificationListener from '../hooks/useNotificationListener';
+import { TabStack } from '../navigation/TabStack';
+import WebviewOverlay from './WebviewOverlay';
+
+export interface AuthenticatedAppProps {
+  initialWer?: string;
```

Comment:
Can we rename all instances of `wer`? To something like `deepLinkPath` or `lurePath` (not totally clear on what it actually is).